src/compass/utils/lut.py

```diff
@@ -183,8 +225,30 @@ def compute_geocoding_correction_luts(burst, dem_path,
     rg_set = resize(rg_set_temp, out_shape, **kwargs)
     az_set = resize(az_set_temp, out_shape, **kwargs)
 
+    # Compute wet and dry troposphere delays using RAiDER
+    wet_los_tropo, dry_los_tropo = [np.zeros(out_shape) for _ in range(2)]
+
+    if weather_model_path is not None:
+        # Instantiate an "aoi" object to read lat/lon/height files
+        aoi = RasterRDR(rdr2geo_raster_paths[1], rdr2geo_raster_paths[0],
+                        rdr2geo_raster_paths[2])
+
+        # Instantiate the Zenith object. Note RAiDER LOS object requires
+        # the orbit files.
+        los = Zenith()
+
+        # Compute the troposphere delay along the Zenith
+        zen_wet, zen_dry = tropo_delay(burst.sensing_start,
+                                       weather_model_path,
+                                       aoi, los)
+
+        # RaiDER delay is one-way only. Get the LOS delay my multiplying
+        # by the incidence angle
+        wet_los_tropo = 2.0 * zen_wet * np.cos(np.deg2rad(inc_angle))
+        dry_los_tropo = 2.0 * zen_dry * np.cos(np.deg2rad(inc_angle))
```

Comment:
`zenith = LOS * cos(incidence) -> LOS = zenith / cos`

```suggestion
        wet_los_tropo = 2.0 * zen_wet / np.cos(np.deg2rad(inc_angle))
        dry_los_tropo = 2.0 * zen_dry / np.cos(np.deg2rad(inc_angle))
```